Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2_test.py

```diff
@@ -939,3 +939,166 @@ def test_fetch_incidents(
         last_run=previous_run,
     )
     assert last_run == expected_last_run
+
+
+def test_check_dictionary_mode_args():
+    """
+    Given:
+     - A mode type ("group" or "machine") and respective arguments (`host_name` and `group_name`).
+    When:
+     - The check_dictionary_mode_args function is called with these arguments.
+    Then:
+     - Ensure the function correctly returns a tuple with the appropriate host name and group name based on the mode.
+    """
+    from CiscoEmailSecurityApplianceIronPortV2 import (
+        check_dictionary_mode_args,
+    )
+
+    assert check_dictionary_mode_args("group", host_name="", group_name="group1") == (None, "group1")
+    assert check_dictionary_mode_args("machine", host_name="host1", group_name="") == ("host1", None)
+
+
+def test_dictionary_list_command(mocker, mock_client):
+    """
+    Given:
+     - A dictionary name as input.
+    When:
+     - The dictionary_list_command is called.
+    Then:
+     - Ensure the command returns the correct information about the dictionary.
+    """
+    from CiscoEmailSecurityApplianceIronPortV2 import (
+        dictionary_list_command,
+    )
+    mocker.patch(
+        "CiscoEmailSecurityApplianceIronPortV2.Client.dictionary_list_request",
+        return_value={
+            "data": [
+                {"name": "Example Dictionary", "details": "Some details"}
+            ]
+        }
+    )
+
+    args = {"dictionary_name": "example_dictionary"}
+    result = dictionary_list_command(mock_client, args)
+
+    assert "Information for Dictionary: example_dictionary" in result.readable_output
```

Comment:
Better test the outputs - HR is not very interesting.

Reply:
Added some asserts for the output.